src/tests/suite/daos_obj.c

```diff
@@ -3748,8 +3748,8 @@ split_sgl_internal(void **state, int size)
 	rc = daos_obj_open(arg->coh, oid, DAOS_OO_RW, &oh, NULL);
 	assert_rc_equal(rc, 0);
 
-	sbuf1 = calloc(size/2, 1);
-	sbuf2 = calloc(size/2, 1);
+	D_ALLOC(sbuf1, size / 2);
+	D_ALLOC(sbuf2, size / 2);
```

Comment:
It's test code, but do these need assert_non_null?

Reply:
In theory, but it'll be easy to tell from any stacktrace if they fail.  The only reason for the change here is the compiler is now warning on mismatched alloc/free calls.